rts/Menu/LuaMenuController.cpp

```diff
@@ -80,6 +79,9 @@ bool CLuaMenuController::Draw()
 	// calls IsAbove
 	mouse->GetCurrentTooltip();
 
+	
+	spring_msecs(10).sleep(true);
```

Comment:
Imo it should be near the step that can take lots of time (drawscreen). But it does not really matter, I was just moving it around to test some things with even bigger delay. Move it wherever you prefer